couchdb/client.py

```diff
@@ -1208,7 +1213,13 @@ def changes(self, **opts):
         """
         if opts.get('feed') == 'continuous':
             return self._changes(**opts)
-        _, _, data = self.resource.get_json('_changes', **opts)
+
```

Comment:
Given this is repeated, maybe it would make sense to extract a helper method here?